src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -273,6 +283,35 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
 
             return searchOptions;
 
+            IEnumerable<IncludeExpression> ParseIncludeIterateExpressions(SearchParams searchParams)
+            {
+                var allIterateModifiers = IncludeIterateModifiers.Concat(RevIncludeIterateModifiers).ToList();
```

Comment:
There's no need to compute this every time

Reply:
Right! Fixed.